src/ossl/aes.rs

```diff
@@ -2065,6 +2164,7 @@ impl MsgEncryption for AesOperation {
         self.msg_encrypt_final(param, paramlen, plain, cipher)
     }
 
+    /// Begin a new message based encryption
```

Comment:
nit: Some of the docs mention the Message Based with upper and some with lower first case. Would be good to have this systematic too.